clients/src/main/resources/common/message/DescribeClientQuotasResponse.json

```diff
@@ -36,7 +36,7 @@
           "about": "The entity name, or null if the default." }
       ]},
       { "name": "Values", "type": "[]ValueData", "versions": "0+",
-	"about": "The quota values for the entity.", "fields": [
+	    "about": "The quota values for the entity.", "fields": [
```

Comment:
The alignment is still not right here.

Reply:
<img width="473" alt="image" src="https://github.com/user-attachments/assets/9140a7f8-d357-43ec-9249-2a8daf608860">
I checked in idea and it looks like the indentation is correct. What should I do?

Reply:
If the alignment is fine, then ignore my comment.

Reply:
I have checked the indentation of all changed files again and modified them. Please help review them again, thank you!